entitlement/src/main/java/org/killbill/billing/entitlement/api/DefaultEntitlement.java

```diff
@@ -402,7 +402,7 @@ public void uncancelEntitlement(final Iterable<PluginProperty> properties, final
         final WithEntitlementPlugin<Void> uncancelEntitlementWithPlugin = new WithEntitlementPlugin<Void>() {
 
             @Override
-            public Void doCall(final EntitlementApi entitlementApi, final EntitlementContext updatedPluginContext) throws EntitlementApiException {
+            public Void doCall(final EntitlementApi entitlementApi, final DefaultEntitlementContext updatedPluginContext) throws EntitlementApiException {
```

Comment:
Why can't we keep the interface?

Reply:
`EntitlementContext` comes from `plugin-api` and doesn't have the setter.

I could have cast'ed, but I prefer being explicit (I almost renamed `DefaultEntitlementContext` as `MutableEntitlementContext` but the changes would be huge).